Assets/Tests/Demo/scripts/verify-replay-via-cli.sh

```
@@ -0,0 +1,205 @@
+
+echo "[6/8] Activating controller + starting replay via CLI..."
+activate_for_replay
+echo "  Starting replay..."
+REPLAY_RESULT=$(uloop replay-input --action Start $PROJECT_OPTS 2>&1) || true
+echo "  $REPLAY_RESULT"
+
+echo "  Waiting for replay to finish..."
+sleep 2
+waited=0
+while [ $waited -lt 60 ]; do
+    STATUS_RESULT=$(uloop replay-input --action Status $PROJECT_OPTS 2>&1) || true
```

Shellcheck (0.11.0)

[warning] Quotes/backslashes in this variable will not be respected.

(SC2090)